beta-3-setup.md

```diff
@@ -3762,6 +3762,10 @@ Running ansible:
     export ROUTE_53_WILDCARD_ZONE=cloudapps.example.com
     export ROUTE_53_HOST_ZONE=example.com
 
+### Bootstrap the playbook directories
+
+    ansible-playbook clone_and_setup_repo.yml
```

Comment:
@detiber Hi Jason, this should probably read `ansible-playbook -i 'localhost,' clone_and_setup_repo.yml`